app/src/main/java/org/mozilla/focus/autocomplete/AutocompleteSettingsFragment.kt

```diff
@@ -41,4 +51,12 @@ class AutocompleteSettingsFragment : PreferenceFragment() {
 
         return super.onPreferenceTreeClick(preferenceScreen, preference)
     }
+
+    override fun onSharedPreferenceChanged(sharedPreferences: SharedPreferences?, key: String?) {
+        if (key == null || sharedPreferences == null) {
+            return
+        }
+
+        TelemetryWrapper.settingsEvent(key, sharedPreferences.all[key].toString())
```

Comment:
Since this is returning a setting value, this will never be something like a custom domain that's saved, right? Just looking at these TelemetryWrapper changes in this patch for data review it's not obvious to me if the custom domains are saved as values in settings (but probably not).

Reply:
Correct. In this screen we only show two switches to enable/disable autocomplete for default and custom domains.
The actual domains are added in another screen and are not stored in the app's default shared preferences (although in a different shared preferences file). Domains are never reported in any of the telemetry probes.